mobile-app/app/screens/AppNavigator/screens/Portfolio/PortfolioScreen.tsx

```diff
@@ -432,16 +453,22 @@ export function PortfolioScreen ({ navigation }: Props): JSX.Element {
       },
       {
         stackScreenName: 'CreateOrEditAddressLabelForm',
-        component: CreateOrEditAddressLabelForm,
+        component: CreateOrEditAddressLabelFormV2,
         option: {
           headerStatusBarHeight: 1,
-          headerBackgroundContainerStyle: tailwind('border-b', {
-            'border-gray-200': isLight,
-            'border-gray-700': !isLight,
-            '-top-5': Platform.OS !== 'web'
-          }),
           headerTitle: '',
-          headerBackTitleVisible: false
+          headerBackTitleVisible: false,
+          headerStyle: tailwind('rounded-t-xl-v2', {
+            'bg-mono-light-v2-100': isLight,
+            'bg-mono-dark-v2-100': !isLight
+          }),
+          headerRight: (): JSX.Element => {
+            return (
+              <ThemedTouchableOpacityV2 style={tailwind('border-none mr-5')} onPress={() => dismissModal(false)}>
```

Comment:
```suggestion
              <ThemedTouchableOpacityV2 style={tailwind('border-none mr-5 mt-2')} onPress={() => dismissModal(false)}>
```

Also can improve spacing on back button - screenshot taken on ios 
![image](https://user-images.githubusercontent.com/13292662/179967191-e84b3d58-2167-4a81-9b25-321d5af33212.png)

Reply:
Adjusted back button by using custom component